tests/template_tests/test_loaders.py

```diff
@@ -87,6 +87,19 @@ def test_load_template_missing(self):
             "Cached loader failed to cache the TemplateDoesNotExist exception",
         )
 
+    @ignore_warnings(category=RemovedInDjango20Warning)
+    def test_load_non_existent_cached_template(self):
+        """
+        #26280 -- Cached template loader crash when loading nonexistent template
+        """
+        loader = self.engine.template_loaders[0]
+
+        with self.assertRaises(TemplateDoesNotExist):
+            loader.find_template('non_existent.html')
```

Comment:
Actually, it is not, because template cache should be filled with `TemplateDoesNotExist` exception, which happens only in `find_template` function